src/BackgroundImage.js

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components'
```

Comment:
As we discussed in person, let's merge the BackgroundImage PR first and see if that clears up some of these unrelated changes

Reply:
I will rebase on master

Reply:
You should be able to pull the latest from master now and merge it into this branch. If you run into really hairy merge conflicts, I can help out